.github/actions/release/action.yml

```diff
@@ -27,7 +27,7 @@ runs:
   using: composite
   steps:
     - name: Setup Node
-      uses: actions/setup-node@v3.6.0
+      uses: actions/setup-node@v4.0.2
```

Comment:
Do we really need the more specific version here? Seems like `@v4` would suffice.